.github/renovate.json5

```diff
@@ -16,14 +16,19 @@
     {
       "customType": "regex",
       "fileMatch": [
-        "^pkg/apis/flux/.*\\.go$"
+        "^pkg/apis/flux/.*\\.go$",
+        "^example/shoot\\.yaml"
       ],
       "matchStrings": [
-        "defaultFluxVersion = \"(?<currentValue>.*?)\"\\n",
+        "defaultFluxVersion = \"(?<currentValue>v[0-9.]+?)\"\\n",
```

Comment:
Are you sure? `v[0-9.]+?` should only match e.g. `v1` in `v1.2.3`, because `?` makes it lazy. I guess `(?<currentValue>v[0-9.]+)` is what we want

Reply:
The regex matches the entire string literal, as it's "anchored" by the trailing double quote.
But you're right, we don't need the lazy quantifier.